go/vt/vtctl/vtctl.go

```diff
@@ -2998,19 +2999,28 @@ func commandValidate(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.
 }
 
 func commandListAllTablets(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
+	keyspaceFilter := subFlags.String("keyspace", "", "Keyspace to filter on")
+	tabletTypeStr := subFlags.String("tablet_type", "", "Tablet type to filter on")
 	if err := subFlags.Parse(args); err != nil {
 		return err
 	}
-
+	var tabletTypeFilter topodatapb.TabletType
+	if *tabletTypeStr != "" {
+		tabletTypeFilter = topodatapb.TabletType(topodata.TabletType_value[strings.ToUpper(*tabletTypeStr)])
+		if tabletTypeFilter == topodatapb.TabletType_UNKNOWN {
+			return fmt.Errorf("invalid tablet type of '%s' specified for -tablet_type", *tabletTypeStr)
+		}
```

Comment:
There is already a function that parses the tablet type in `topoproto` package called `ParseTabletType`. Maybe we should use that here?

Reply:
Actually there is also a local function in `vtctl.go` called `parseTabletType` which calls the above mentioned function internally, and it used like `parseTabletType(subFlags.Arg(1), topoproto.AllTabletTypes)`

Reply:
Ah, nice! I didn't even notice that. Thanks! Using that function now.